### PR TITLE
[OSCD] Added a rule to detect potential persistence using registry keys

### PR DESCRIPTION
**Pull request by V3T0**
Added a rule to detect persistence by using the registry key: HKLM\SOFTWARE\Microsoft\Active Setup\Installed Components. An attacker can add payload in StubPath value and execute it via runonce.exe

### CODE REVIEW

File: rules/windows/process_creation/win_susp_tracker_execution.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title: DLL Injection with Tracker.exe
+id: 148431ce-4b70-403d-8525-fcc2993f29ea
+description: This rule detects DLL injection and execution via LOLBAS - Tracker.exe
+author: Avneet Singh '@v3t0_', oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_susp_tracker_execution.yml:4]: could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?

**Comment by aw350m33d** [comment]
Hello Avneet! We are very grateful to you for participating in the initiative! 👍🏻 

It seems you created a branch, not from the `oscd` branch. Because of this, there are a lot of unnecessary changes to other files in your pull request. 😿  I ask you to fix this. 🙏🏻 

You can either do this on the current branch and, accordingly, on the current pull request. Or create a new branch from the `oscd` branch and apply necessary changes to it. 

This will help us a lot when we do the merge. Thank you!

**Comment by V3T0** [comment]
> 
> 
> Hello Avneet! We are very grateful to you for participating in the initiative! 👍🏻
> 
> It seems you created a branch, not from the `oscd` branch. Because of this, there are a lot of unnecessary changes to other files in your pull request. 😿 I ask you to fix this. 🙏🏻
> 
> You can either do this on the current branch and, accordingly, on the current pull request. Or create a new branch from the `oscd` branch and apply necessary changes to it.
> 
> This will help us a lot when we do the merge. Thank you!

Hi @aw350m33d, thanks for providing me the suggestions, I really appreciate. :) 
Yes I realized that there are multiple commits in my all of the PRs. I followed the following process:
Step1: Forked sigma repo
Step2: Cloned the forked repo on my local system
Step3: Made a new branch, did changes and pushed the changes to the locally forked project.
Step4: Issues a Pull Request from GitHub Web by selecting destination as OSCD branch.

Is there anything wrong that I am doing in while issuing the Pull Request?

I would appreciate your help on this. Thanks in advance :)

**Comment by V3T0** [comment]
@aw350m33d I think I have figured it out why this happened. I will close this PR and other 2 faulty ones and will issue a new PR. :)

**Comment by V3T0** [comment]
closing this faulty PR as it contains other commits as well.

Opened a new PR: #1235

**Pull request closed without merging**